src/coreclr/nativeaot/Runtime/GCMemoryHelpers.inl

```diff
@@ -156,6 +156,10 @@ static const uint32_t INVALIDGCVALUE = 0xcccccccd;
 
 FORCEINLINE void InlineWriteBarrier(void * dst, void * ref)
 {
+#ifdef TARGET_WASM
```

Comment:
This does not need to be under wasm ifdef.

Reply:
thanks